### ISSUE TITLE
Get an error "Flutter plugin not installed" while i have installed it. Suggestion code not working

### ISSUE THREAD

**Issue by pqtrong17** [flutter/flutter#70601]
Firstly, sorry because my English is not good!

``flutter doctor -v :``
```
[√] Flutter (Channel stable, 1.22.3, on Microsoft Windows [Version 10.0.18363.1198], locale en-US)
    • Flutter version 1.22.3 at C:\Users\pqtrong17\AppData\Local\Android\flutter
    • Framework revision 8874f21e79 (3 weeks ago), 2020-10-29 14:14:35 -0700
    • Engine revision a1440ca392
    • Dart version 2.10.3


[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\pqtrong17\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[!] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (1 available)
    • SM N950U (mobile) • 192.168.1.43:5555 • android-arm64 • Android 9 (API 28)

! Doctor found issues in 1 category.

```
Although I have installed ``dart`` and ``flutter`` plugin but IDE get an error and suggestion code not working (Ctrl + Space not working).

**Comment by pedromassangocode** [comment]
Hi @pqtrong17  
This is a known issue, it happens because the latest version of Android Studio changed the path where the plugins are located and Flutter tools can't find it. This is just a warning and you can still use Flutter without worries.

Thank you

**Comment by pqtrong17** [comment]
> Hi @pqtrong17
> This is a known issue, it happens because the latest version of Android Studio changed the path where the plugins are located and Flutter tools can't find it. This is just a warning and you can still use Flutter without worries.
> 
> Similar to #70551.
> 
> Thank you

But this make suggestion code not working. How to fix it? I not found solution on #70551 ?

**Comment by pedromassangocode** [comment]
> But this make suggestion code not working. How to fix it? I not found solution on #70551 ?

This has nothing to do with code suggestion (intellisense). You may be having a issue unrelated to this  one, please open a issue in the Plugin repository [repository](https://github.com/flutter/flutter-intellij/issues)
